partcad/src/partcad/project_factory_git.py

```
@@ -86,82 +115,109 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
+                                attempt + 1,
+                                max_retries,
+                                patience,
+                            )
+                            time.sleep(patience)
+                        else:
+                            pc_logging.error("Failed to update repo after %s retries" % attempt)
+                            # Fall back to using the previous copy
+                else:
+                    # Clone the repository if it's not cached yet.
+                    try:
+                        pc_logging.info("Cloning the GIT repo: %s" % self.import_config_url)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)